Build/validate-qir.ps1

```diff
@@ -57,23 +71,24 @@ function Build-One {
 
 # The commented out lines are sample projects that are not yet compatible for QIR generation/execution. 
 # 'not compatible' means that the structure of the sample is not compatible for QIR generation.
-# 'needs argument(s)' means that the sample can generated QIR, but running the exe requires command line arguments.
+# 'relies on hardware' means that the sample targets a hardware architecture that is not yet supported for QIR.
+# 'memory leak' means that the sample stalls when running and uses way too many system resources.
 $qirProjects = @(
     # not compatible #@{ Path = (Join-Path $PSScriptRoot .. samples algorithms chsh-game CHSHGame.csproj); Args = @() },
     # not compatible #@{ Path = (Join-Path $PSScriptRoot .. samples algorithms database-search DatabaseSearchSample.csproj); Args = @() },
     # not compatible #@{ Path = (Join-Path $PSScriptRoot .. samples algorithms integer-factorization IntegerFactorization.csproj); Args = @() },
     @{ Path = (Join-Path $PSScriptRoot .. samples algorithms oracle-synthesis OracleSynthesis.csproj); Args = @() },
     @{ Path = (Join-Path $PSScriptRoot .. samples algorithms order-finding OrderFinding.csproj); Args = @("--index", "1") },
-    @{ Path = (Join-Path $PSScriptRoot .. samples algorithms repeat-until-success RepeatUntilSuccess.csproj); Args = @("--gate", "simple", "--input-value", "true", "--input-basis", "PauliX", "--limit", "4", "--num-runs", "2") },
+    @{ Path = (Join-Path $PSScriptRoot .. samples algorithms repeat-until-success RepeatUntilSuccess.csproj); Args = @("--gate", "simple", "--inputValue", "true", "--inputBasis", "PauliX", "--limit", "4", "--numRuns", "2") },
```

Comment:
Perhaps tangential, but why are the input parameter names different here than what's generated for stand-alone executables?

Reply:
I noticed this as well, part way through doing this. I'm not sure what the reason is for the change in case for parameter names. @cesarzc might know more.

Reply:
No reason, just a matter of us properly aligning all APIs.